src/qml/imports/Theme/QfSlider.qml

```diff
@@ -0,0 +1,57 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+
+Item {
+    property alias from: slider.from;
+    property alias to: slider.to
+    property alias stepSize: slider.stepSize
+    property alias value: slider.value
+    property string prefixText: ""
+    property string suffixText: ""
+
+    signal moved
+
+    id: wrapper
+
+    implicitHeight: 10
+
+    RowLayout {
+        id: layout
+        anchors.fill: parent
+
+        Slider {
+          Layout.fillWidth: true
+          id: slider
+          value: 50
+          from: 0
+          to: 100
+          stepSize: 1
+          onMoved: wrapper.moved()
+        }
+
+        Label {
+            id: suffix
+            text: prefixText + value + suffixText
+
+            Layout.fillWidth: true
```

Comment:
I don't think you should fillWidth here, we should have the slider take as much space as possible.